scripts/utils/compile-babel.js

```diff
@@ -14,20 +14,26 @@ function getCommand(watch, dir) {
 
   const args = [
     './src',
-    `--out-dir ${dir}`,
-    `--config-file ${path.resolve(__dirname, '../../.babelrc.js')}`,
-    `--copy-files`,
+    `--out-dir=${dir}`,
+    `--config-file=${path.resolve(__dirname, '../../.babelrc.js')}`,
   ];
 
+  // babel copying over files it did not parse is a anti-pattern
+  // but in the case of the CLI, it houses generators are are templates
+  // moving all these is a lot of work. We should make different choices when we eventually refactor / rebuild the CLI
+  if (process.cwd().includes(path.join('lib', 'cli'))) {
+    args.push('--copy-files');
+  }
```

Comment:
Only add the `--copy-files` flag for `lib/cli`.

Everywhere else, I've moved files to the root of the package and updated references; but in this package, it's quite hard to do. The generators are all none-source files, but they are currently placed in the `src` directory.

We've been talking about overhauling the `lib/cli` anyway: https://github.com/storybookjs/storybook/issues/1108
So it's OK if we leave this as-is for now.